Docs for the 0.22 release (#2307)
kestra-io/docs: content/docs/configuration/index.md

    org.apache.kafka: DEBUG
    io.netty.handler.logging: TRACE
```
To disable logging to the server logs, you can configure the following:

```yaml
logger:
  levels:
    flow: 'OFF'
```

> This disables all flow execution logs to go to the server logs. Similarly, the following configuration disables the execution logs for the `hello-world` flow:

Reviewer: ```suggestion
This disables all flow execution logs from being stored so that those logs will only show up in the server logs. 

Similarly, the following configuration disables the execution logs for the `hello-world` flow:
```

          taskRunner:
            type: io.kestra.plugin.scripts.runner.docker.Docker
```
### Plugin Management

As of Kestra 0.22.0, you can configure Plugins management properties for things like remote storage, custom plugins, and versioned plugins. An example configuration for managing plugins looks as follows:

```yaml
kestra: 
  plugins:
      management:
        enabled: true # setting to false will make Versioned plugin tab disappear + API will return an error
        remoteStorageEnabled: true
        customPluginsEnabled: true # setting to false will disable installing or uploading custom plugins
        localRepositoryPath: /tmp/kestra/plugins-repository
        autoReloadEnabled: true
        autoReloadInterval: 60s
        defaultVersion: LATEST 
```

The following list of properties correspond to Versioned and remotely stored plugins:

> - `remoteStorageEnabled`: Specifies whether remote-storage is enabled (i.e., plugins are stored on the internal storage).
> - `localRepositoryPath`: The local-path where managed plugins will be synced.

Reviewer: ```suggestion
- `remoteStorageEnabled`: Specifies whether remote storage is enabled (i.e., plugins are stored on the internal storage).
- `localRepositoryPath`: The local path where managed plugins will be synced.
```

          taskRunner:
            type: io.kestra.plugin.scripts.runner.docker.Docker
```
### Plugin Management

> As of Kestra 0.22.0, you can configure Plugins management properties for things like remote storage, custom plugins, and versioned plugins. An example configuration for managing plugins looks as follows:

Author: ```suggestion
As of Kestra 0.22.0, you can configure plugin management properties for remote storage and custom and versioned plugins. An example configuration for managing plugins looks as follows:
```

## Secret Managers

You can configure the [secret manager](../06.enterprise/02.governance/secrets-manager.md) backend using the `kestra.secret` configuration. To isolate your secret manager from specific [Kestra services](../07.architecture/02.server-components.md), you can specify the following in your configuration using `kestra.secret.isolation`:

```yaml
kestra:
  secret:
    type: azure-key-vault
    azureKeyVault:
      clientSecret:
        tenantId: "id"
        clientId: "id"
        clientSecret: "secret"

>     isolation: # new

Reviewer: ```suggestion
    isolation: # available in Kestra >=0.22
```
just saying "new" will age too quickly :)

Author: That was a copy paste mistake, good catch 🙈

- [Storage Minio](#minio) compatible with  others *S3 like* storage services
- [Storage Azure](#azure) for [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/)

To isolate your dedicated internal storage from specific [Kestra services](../07.architecture/02.server-components.md), you can specify the following in your configuration using `kestra.storage.isolation`:

```yaml
kestra:
  storage:
    type: gcs

>     isolation:

Reviewer: ```suggestion
    isolation: # available in Kestra >=0.22
```
